provisioning_client/inc/azure_prov_client/internal/prov_transport_utils.h

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#ifndef PROV_TRANSPORT_UTIL_H
+#define PROV_TRANSPORT_UTIL_H
+
+#ifdef __cplusplus
+extern "C" {
+#include <cstdint>
+#else
+#include <stdint.h>
+#endif /* __cplusplus */
+
+uint32_t parse_retry_after_value(const char* retry_after);
```

Comment:
We don't need a new header for this. We could accomplish the same in prov_transport.h

Reply:
The source will need to live somewhere.  prov_transport.h doesn't have a matching source file.

---
In reply to: [280640096](https://github.com/Azure/azure-iot-sdk-c/pull/984#discussion_r280640096) [](ancestors = 280640096)

Reply:
Moving the code to an h only situation causes unintended consequences.  The Unittests have issues with doubly defined symbols.

---
In reply to: [281403483](https://github.com/Azure/azure-iot-sdk-c/pull/984#discussion_r281403483) [](ancestors = 281403483,280640096)

Reply:
static inline should work, the compiler will generate just one instance of it and you can just include the header to point to that address

---
In reply to: [281726248](https://github.com/Azure/azure-iot-sdk-c/pull/984#discussion_r281726248) [](ancestors = 281726248,281403483,280640096)